components/AnchorNavigation/src/index.scss

```diff
@@ -0,0 +1,46 @@
+.denhaag-anchor-navigation {
+  border-inline-start: var(--denhaag-anchor-navigation-border-inline-start);
+}
+
+.denhaag-anchor-navigation ul {
+  list-style: var(--denhaag-anchor-navigation-ul-list-style);
+  padding-inline-start: var(--denhaag-anchor-navigation-ul-padding-inline-start);
+}
+
+.denhaag-anchor-navigation ul li a {
```

Comment:
This kind of nesting is unnecassary,  `.denhaag-anchor-navigation ul li a` should do the same as `.denhaag-anchor-navigation a`

Or if u never want any issue with it apply a CSS selector: `.denhaag-anchor-navigation__link`